python/lsst/cp/pipe/cpCertify.py

```diff
@@ -69,19 +75,78 @@ def __init__(self, *, registry, inputCollection, outputCollection, lastRunOnly=T
         self.inputCollection = inputCollection
         self.outputCollection = outputCollection
 
-    def run(self, datasetTypeName, timespan):
+    def run(self, datasetTypeName, beginDate=None, endDate=None):
```

Comment:
If you prefer run to take a Timespan it still can. You don't need to change the `run` method here -- it's allows for the `main()` below to convert the arguments to a time span.  It just depends on what you feel is best for this API.